libmscore/read114.cpp

```diff
@@ -952,6 +952,32 @@ static void readTuplet(Tuplet* tuplet, XmlReader& e)
             }
       }
 
+//---------------------------------------------------------
+//   readTremolo
+//---------------------------------------------------------
+
+static void readTremolo(Tremolo* tremolo, XmlReader& e)
+      {
+      while (e.readNextStartElement()) {
+            if (e.name() == "subtype") {
+                  int sti = e.readElementText().toInt();
+                  TremoloType st;
+                  switch (sti) {
+                        default:
+                        case 0: st = TremoloType::R8;  break;
```

Comment:
I would create local enum with the values like TremoloType::OLD_R8, TremoloType::OLD_R16, etc. to avoid using magic numbers here. And add comment like "the switch allows compatibility with old tremolo 1.X format"

Reply:
Ok done.